src/Coin.cpp

```diff
@@ -195,14 +196,20 @@ std::string TW::deriveAddress(TWCoinType coin, const PublicKey& publicKey) {
     case TWCoinTypeBravoCoin:
         return Bravo::Address(publicKey).string();
 
+    case TWCoinTypeCosmos:
+        return Cosmos::Address(HRP_COSMOS, publicKey).string();
+
     case TWCoinTypeDash:
         return Bitcoin::Address(publicKey, TWP2PKHPrefixDash).string();
 
     case TWCoinTypeDecred:
         return Decred::Address(publicKey).string();
 
     case TWCoinTypeDogecoin:
-        return Bitcoin::Address(publicKey, TWP2PKHPrefixDogecoin).string();
+        return Bitcoin::Address(publicKey, TWP2PKHPrefixD).string();
+
+    case TWCoinTypeDigiByte:
+        return Bitcoin::SegwitAddress(publicKey, 0, HRP_DIGIBYTE).string();
```

Comment:
I think most exchanges do not support bech32 address for DG, should we use `Bitcoin::Address(publicKey, TWP2PKHPrefixD).string();` instead? 

@hewigovens what do you think?

Reply:
I suggest to ask DigiByte team. GRS/VIA want bip84 and Qtum wants bip44

Reply:
Fair point, actually DigiByte guys suggested BIP84 when I talked to them few days ago